src/harrastuspassi/harrastuspassi/api.py

```diff
@@ -75,10 +78,62 @@ def filter(self, qs, value):
         return super().filter(qs, list(values_with_children))
 
 
+class NearestOrderingFilter(filters.OrderingFilter):
+    def get_coordinates(self):
+        errors = defaultdict(list)
+        try:
+            near_latitude = self.parent.data.get('near_latitude')
+        except KeyError:
+            errors['near_latitude'].append(_('This field is required when nearest ordering is used.'))
+        try:
+            near_longitude = self.parent.data.get('near_longitude')
+        except KeyError:
+            errors['near_longitude'].append(_('This field is required when nearest ordering is used.'))
+        try:
+            near_latitude = float(near_latitude)
+        except ValueError:
```

Comment:
This can also raise `TypeError` if `near_latitude` is `None`.

Reply:
Also if the code above is changed so that `KeyError` is raised if the value is not found, then `near_latitude` can be undefined here.